data/json/items/gun/44paper.json

```diff
@@ -4,8 +4,16 @@
     "copy-from": "pistol_revolver_cap_ball",
     "looks_like": "sw_619",
     "type": "GUN",
-    "name": { "str_sp": "Colt M1860 Army" },
+    "name": { "str_sp": "civil war revolver" },
     "description": "The Colt Army Model 1860 is a cap & ball .44-caliber revolver.  It was used during the American Civil War, and made by Colt's Manufacturing Company.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "colt_army",
+        "name": { "str": "Colt M1860 Army", "str_pl": "Colt M1860 Armies" },
+        "description": "The Colt M1860 is a black powder, cap & ball revolver dating back to the Civil War.  While reloading it takes quite awhile due to having to put a primer on each chamber manually, instead of having the primer in one unit with the casing, like metallic cartridges.  However, for the time 6 shots in quick succession was a great thing for cavalry and officers to have on the field.  Nowadays, this firearm is completely outdated, and only sees use in collections, reenactment, or special shooting competitions.  However, black powder guns such as this are less regulated than more modern firearms, so it is sometimes the first gun a youth may buy.  This one is chambered in .44, and while the number may remind some of .44 Magnum, it has nowhere near the ballistic effectiveness of .44 Magnum."
```

Comment:
```suggestion
        "description": "The Colt M1860 is a black powder, cap & ball revolver dating back to the Civil War.  While reloading it takes quite a while due to having to put a primer on each chamber manually, instead of having the primer in one unit with the casing, like metallic cartridges.  However, for the time 6 shots in quick succession was a great thing for cavalry and officers to have on the field.  Nowadays, this firearm is completely outdated, and only sees use in collections, reenactment, or special shooting competitions.  However, black powder guns such as this are less regulated than more modern firearms, so it is sometimes the first gun a youth may buy.  This one is chambered in .44, and while the number may remind some of .44 Magnum, it has nowhere near the ballistic effectiveness of .44 Magnum."
```